CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 All notable changes to the "azurefunctions" extension will be documented in this file.
 
+## 0.9.0 - 2018-05-09
+
+### Added
+
+- Improved warning when Azure Functions Core Tools is out of date
+  - In some cases, an 'Update' option is provided that will automatically run the necessary commmands
+  - Warning now displays for all users, regardless of how they installed the func cli
+- Function templates are now versioned and can be controlled with the "azureFunctions.templateVersion" setting
+- Load more repositories when configuring deployment source to GitHub
```

Comment:
Could consider this a bug and leave it out.

Reply:
No preference on my side. I'll remove if that's what you want